packages/framework/src/Form/Admin/Administrator/AdministratorFormType.php

```diff
@@ -160,6 +160,35 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'back_route' => 'admin_administrator_list',
                 'entity' => $adminToEdit,
             ]);
+
+        if ($options['scenario'] !== self::SCENARIO_EDIT || !$this->canWorkWithRoles($adminToEdit) || !$this->getCurrentAdministrator()->isSuperadmin()) {
+            return;
+        }
+
+        $builderPromoteGroup = $builder->create('promote', GroupType::class, [
+            'label' => 'Super Administrator',
+        ]);
+
+        $builderPromoteGroup->add('promoteToSuperadmin', DisplayOnlyUrlType::class, [
```

Comment:
This is weird to use `DisplayOnlyUrlType` and show something totally different in that type :D

Reply:
It only displays the url that promotes user to superadmin 😆